### DISCUSSION TITLE
Adding button translate messages

### DISCUSSION THREAD

**Discussion by AdrienSama** [bluesky-social/atproto#1746] [Bluesky Client App]
With an addition of button, our could translate others posts of anothers person... 🤷🏻‍♂️

**Comment by bnewbold** [comment]
If you click on the three-button icon under a post, there should be a "Translate" option. Does that work for you?

![2023-10-23T12:04:57,007162626-04:00](https://github.com/bluesky-social/atproto/assets/130077/6dd47e59-60b9-47fc-96da-51d4a968ed02)

**Reply by venteto** [reply]
I just noticed today in a thread of replies that are all in Italian, the, er, focused (?) / selected (?) / highlighted (?) reply has a translation link not buried behind any additional mouse clicks, whereas all the other replies have to be manually selected, one by one, to reveal the same link.

It would be a nice touch if _every post_ had the same non-hidden translate link, _without any extra intervention_ by the user (I mean in lieu of some later in-app automatic translation feature for all posts).

![bluesky-translate-link](https://github.com/bluesky-social/atproto/assets/7062385/70ef916f-3fbe-46f9-8fd9-9a4fc02796a6)

**Comment by Zero3K** [comment]
Speaking of the translate button, it should use DeepL instead of Google Translate.

**Comment by AdrienSama** [comment]
Yes, i know ! This message is of the long date. But, thanks !
